readme.txt

```diff
@@ -2,50 +2,51 @@
 Requires at least: 2.9.2
-Tested up to: 5.4.2
-Stable Tag: 1.7.17
+Tested up to: 5.5.1
+Stable Tag: 2.0.0
 License: GPLv2 or later
 License URI: http://www.gnu.org/licenses/gpl-2.0.html
 
 Ecommerce Analytics and behaviour-driven customer engagement tools for ecommerce brands.
 
 == Description ==
 
-Metrilo is the growth marketing platform for modern consumer brands.
-Metrilo’s analytics measure your business performance and give you insights to make strategic decisions. The engagement tools build on top of the vast data made accessible and enable you to bond with your customers long-term.
+[Metrilo](https://www.metrilo.com/?ref=wpplugin) is a growth platform for the modern product brands with a long-term strategy and looking for profitability. Metrilo offers tools for marketing optimization, customer management, engagement and retention as well as email and feedback gathering automation. It facilitates growth through usable data and unique focus on driving repeat sales. Tailored to be used by brand founders, agencies and small marketing teams.
 
-Metrilo helps your brand grow by making your data actionable and increasing customer retention.
+The extension connects easily to your Woocommerce store and processes sales and marketing data into ready-to-use reports and rich customer insights, including historical data - Metrilo will make past orders and old customers accessible for reactivation, too! Then, you can monitor how your store is doing via the dashboard with all key metrics in real time. The Marketing performance, Revenue breakdown and Product performance reports show opportunities for optimization. The Customer Database makes your first-party customer data usable for segmentation and engagement via emails or retargeting. The Customer Retention feature helps you understand the customer lifecycle and loyalty to drive more repeat sales. And finally, the emails - manual and automated, utilize all data to reach customers at the right time with the right message. Metrilo opens up opportunities to grow your revenue without spending more on marketing.
 
-= Optimize your WooCommerce store marketing =
-Metrilo connects marketing and sales to show you how to maximize ROI
+= Account & Pricing =
+An account is required to use the Metrilo extension. For a free trial, please visit our [sign up page](https://www.metrilo.com/signup?ref=wpplugin). Additional fees apply. Pricing is available on our [pricing page](https://www.metrilo.com/pricing?ref=wpplugin).
 
-* Key sales metrics and marketing performance at a glance
-* Comparison of marketing channels
-* Real-time revenue reporting and many breakdowns
-* Pre-set sales funnel tracking
-* Product performance data to help you develop your brand
+= Features: =
+* Sales performance dashboard - all essential KPIs up-to-date in real time
```

Comment:
After the latest changes in our navigation we have a bit different naming for our features. Maybe it's a bit misleading to keep the text as is?
```
text                            navigation

Revenue breakdown               Analytics > Revenue
Product performance             Analytics > Products
Customer Retention              Analytics > Retention

Sales performance dashboard     What would that be in navigation?
```